docs/iris/src/whatsnew/contributions_1.9/deprecate_2015-11-01_unit-module.txt

```diff
@@ -0,0 +1 @@
+* deprecated :mod:`iris.unit`
```

Comment:
"Yeah sorry guys we've decided we think units are a silly idea after all..."

:+1: for that, I'll add it in.